src/web/routes/application/steps/address/select-address/select-address.js

```diff
@@ -33,14 +35,24 @@ const resetAddressState = (req) => {
   }
 }
 
-const behaviourForGet = (config, journey) => (req, res, next) => {
+const getAddressDataFromSession = (req, addressId) => {
+  const addresses = req.session.postcodeLookupResults.map(buildAddressOption(addressId))
```

Comment:
You could inline this on line 41

Reply:
it's also referenced on line 42, so I don't think I can?

Reply:
oh, i didn't see that